test/test_other.py

```diff
@@ -8998,6 +8998,19 @@ def test_exceptions_rethrow_stack_trace_and_message(self, wasm_eh):
                       expected_output=rethrow_stack_trace_checks, regex=True)
     self.assertNotContained('important_function', err)
 
+  @parameterized({
+    '': (False,),
+    'wasm': (True,),
+  })
+  def test_exceptions_exit_runtime(self, wasm_eh):
+    self.set_setting('EXIT_RUNTIME')
+    if wasm_eh:
+      self.require_wasm_eh()
+      self.emcc_args.append('-fwasm-exceptions')
+    else:
+      self.set_setting('DISABLE_EXCEPTION_CATCHING', 0)
```

Comment:
Does `@with_both_eh_sjlj` not do what you want here?

Reply:
I have a followup and uses that here, and in other places in this file, but for now I'm just following to existing pattern of other exception handling tests in this file.

(BTW the followup has to also move `with_both_eh_sjlj` to common.py so its non-trivial thing and I want to keep this PR focused).